swarm/multihash/README_SWARM.md

```diff
@@ -1,5 +0,0 @@
-# go-multihash library packaged with Swarm
-
-We packaged the go-multihash library with Swarm, because we modified it to add support for BMT
```

Comment:
when we don't vendor it we kind of lock ourselves out of having updates through `govendor`. i'm not sure that pulling it into the source code was the right decision in the first place

Reply:
but we don't use this anymore now, right?